packages/realtime_client/lib/src/realtime_channel.dart

```diff
@@ -248,12 +248,47 @@ class RealtimeChannel {
         (reason, [ref]) => callback(reason?.toString()));
   }
 
-  RealtimeChannel on(
-    RealtimeListenTypes type,
-    ChannelFilter filter,
-    BindingCallback callback,
-  ) {
-    return onEvents(type.toType(), filter, callback);
+  RealtimeChannel onPostgresChanges({
+    required PostgresChangeEvent event,
+    String? schema,
+    String? table,
+    PostgresChangeFilter? filter,
+    required void Function(PostgresChangePayload payload) callback,
+  }) {
+    return onEvents(
+      'postgres_changes',
+      ChannelFilter(
+        event: event.toRealtimeEvent(),
+        schema: schema,
+        table: table,
+        filter: filter?.toString(),
+      ),
+      (payload, [ref]) => callback(PostgresChangePayload.fromPayload(payload)),
```

Comment:
Previously, the user was able to obtain `ref`. This is now gone. I'm wondering if there is any real use case for it.

Reply:
Good point. Honestly not sure how the `ref` can be used as a developer, but the js SDK doesn't have it anymore, so we should be good to remove it. 
https://github.com/supabase/realtime-js/blob/master/src/RealtimeChannel.ts#L350